auth/handler.go

```diff
@@ -208,16 +208,35 @@ func (h *Handler) FindAllUserSessions(ctx context.Context, userID string) ([]Use
 	return sessions, nil
 }
 
-// ServeLogout will clear the current session cookie and end the session (if any).
+// ServeLogout will clear the current session cookie and end the session(s) (if any).
 func (h *Handler) ServeLogout(w http.ResponseWriter, req *http.Request) {
-	h.setSessionCookie(w, req, "")
+	ClearCookie(w, req, CookieName)
+	var sessionIDs []string
+	for _, c := range req.Cookies() {
+		if c.Name != CookieName && c.Name != v1CookieName {
+			continue
+		}
```

Comment:
should this instead be
```suggestion
		if c.Name != CookieName || c.Name != v1CookieName {
			continue
		}
```

Reply:
gotcha, in my head i read it as maybe needing to be "if the current version cookie name isn't present, or the old version isn't present, continue/skip processing"